src/adr/0064-account-delegation.md

```diff
@@ -0,0 +1,48 @@
+# 64. Account Delegation
+
+Date: 2022-11-15
+
+## Status
+
+Accepted (lazy consensus).
+
+Not yet implemented.
```

Comment:
Please states that this relates to https://github.com/apache/james-project/blob/master/src/adr/0061-delegation.md